ContratApp/android/app/google-services.json

```diff
@@ -0,0 +1,47 @@
+{
+  "project_info": {
+    "project_number": "797435410304",
+    "project_id": "contratapp-749e8",
+    "storage_bucket": "contratapp-749e8.appspot.com"
+  },
+  "client": [
```

Comment:
I'd go with an "Web App" approach instead of the Android app due to the setup, its necessary to hide this file with env variables for safety